src/dsp_tools/commands/xmlupload/models/ontology_diagnose_models.py

```diff
@@ -41,14 +46,18 @@ def execute_problem_protocol(self) -> tuple[str, pd.DataFrame | None]:
             the error message and a dataframe with the errors if they exceed 50 or None
         """
         extra_separator = "\n\n---------------------------------------\n\n"
-        msg = "\nSome property and/or class type(s) used in the XML are unknown:" + extra_separator
+        msg = (
+            f"\nSome property and/or class type(s) used in the XML are unknown.\n"
+            f"The ontologies for your project on the server are:{list_separator}"
+            f"{list_separator.join(self.ontos_on_server)}{extra_separator}"
+        )
         cls_msg = self._compose_problem_string_cls()
         if cls_msg:
             msg += cls_msg + extra_separator
         prop_msg = self._compose_problem_string_props()
         if prop_msg:
             msg += prop_msg
-        if len(self.classes) + len(self.properties) > 50:
+        if len(self.classes) + len(self.properties) > 20:
```

Comment:
I think that's a too low number. It's easily possible to scroll through 50 lines in a terminal. I would be annoyed to have an unknown Excel file in my folder for such an unimportant reason.